### PR TITLE
AP_HAL_ChibiOS:add back displayport for KakuteF7

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Comment by IamPete1** [comment]
Any idea how much flash this costs? How I guess plane is the closest, how much are we left with?

**Comment by Hwurzburg** [comment]
about 10K of the 36K for Plane and 60K for Copter....the KakuteF4 has it and has about 26K left also
something seems broken in the size check it does not compare to master anymore

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat [hunk 1]
```diff
@@ -158,8 +158,13 @@ define HAL_PARACHUTE_ENABLED 0
 define HAL_SPRAYER_ENABLED 0
 
 # save FLASH, but leave above when flash issue is fixed
+
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat:161]:
```suggestion
```

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat [hunk 2]
```diff
@@ -158,8 +158,13 @@ define HAL_PARACHUTE_ENABLED 0
...
 include ../include/minimize_features.inc
 
+undef HAL_WITH_MSP_DISPLAYPORT
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat:164]: Comment here

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat [hunk 3]
```diff
@@ -157,9 +157,12 @@ define AP_BATTMON_SMBUS_ENABLE 0
 define HAL_PARACHUTE_ENABLED 0
 define HAL_SPRAYER_ENABLED 0
 
-# save FLASH, but leave above when flash issue is fixed
+# save FLASH, but make sure MSP HD Goggle support remains
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat:160]:
```suggestion
# save FLASH, but leave above when flash issue is fixed
```

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat [hunk 4]
```diff
@@ -157,9 +157,12 @@ define AP_BATTMON_SMBUS_ENABLE 0
...
 include ../include/minimize_features.inc
-
+undef HAL_WITH_MSP_DISPLAYPORT
+define HAL_WITH_MSP_DISPLAYPORT 1
+undef HAL_MSP_ENABLED
+define HAL_MSP_ENABLED 1
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat:165]:
```suggestion

# make sure MSP HD Goggle support remains
undef HAL_WITH_MSP_DISPLAYPORT
define HAL_WITH_MSP_DISPLAYPORT 1
undef HAL_MSP_ENABLED
define HAL_MSP_ENABLED 1
```

**Review by peterbarker — APPROVED** [review]

**Pull request merged**